src/Apps/Browser/Pages/Models.elm

```diff
@@ -35,7 +35,7 @@ type Model
 
 
 initialModel : Web.Site -> Model
-initialModel { url, type_ } =
+initialModel ({ url, type_ } as site) =
```

Comment:
site includes unnecessary type, page doesn't need to receive this intel, better to it receive only ```url content meta``